packages/frontend/src/app/api/(internal)/discolupe/route.ts

```diff
@@ -64,8 +64,10 @@ function toResponseProject(
     tvs,
     display: project.display,
     type: project.type === 'layer2' ? 'L2' : 'L3',
-    arePermissionsDiscoveryDriven: arePermissionsDiscoveryDriven(project),
-    areContractsDiscoveryDriven: areContractsDiscoveryDriven(project),
+    arePermissionsDiscoveryDriven: arePermissionsDiscoveryDriven(
+      project.permissions,
+    ),
+    areContractsDiscoveryDriven: areContractsDiscoveryDriven(project.contracts),
```

Comment:
Can we somehow get rid of this? Maybe add this in runConfigAdjustments? And then we can stop exporting it from config.